packages/e2e-test-utils-playwright/src/blocks.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * WordPress dependencies
+ */
+import { addQueryArgs } from '@wordpress/url';
+
+/**
+ * Delete all blocks using REST API.
+ *
+ * @this {import('.').TestUtils}
+ */
+export async function deleteAllBlocks() {
```

Comment:
This util is originally `trashAllPosts('wp_block')`, altered to use `rest` and `batch` instead to speed it up.